includes/class-wp-job-manager-data-cleaner.php

```diff
@@ -169,6 +169,16 @@ class WP_Job_Manager_Data_Cleaner {
 		'_company_video',
 	);
 
+	/**
+	 * Widget options to be deleted.
+	 *
+	 * @var $widget_options
+	 */
+	private static $widget_options = array(
+		'widget_widget_featured_jobs',
```

Comment:
These can probably just be added to the `$options` array above instead of creating one specifically for widget options.

Reply:
Hey @jom! Thanks for the suggestion! It's done... Please review it and if there are any changes left to do, let me know and I'll gladly make them!